templates/index.html

```diff
@@ -138,6 +140,17 @@
         </paper-toolbar>
         <div id="container" class="container" layout="" vertical="" center="">
           <x-schedule id="mainschedule" show="all"></x-schedule>
+          <div id="GithubUpdateBox" style="text-align:center; background-color:white;">
+            <p style="padding:12px; margin-bottom:12px; margin-top:0px; font-size:1.4rem; font-weight:300;">
+              <a style="color:black; text-decoration:none;" target="_blank">
```

Comment:
If you use an anchor (hyperlink) tag here, might as well set the `href` property to point to something, e.g., https://github.com/EastsidePreparatorySchool/epschedule/commits/master.

Reply:
ping